blog/_posts/2017-08-09-two-girls-meet-in-shanghai.md

```diff
@@ -0,0 +1,57 @@
+---
+title: So, two girls meet in Shanghai…?
+layout: post
+created_at: Wednesday August 09 2017
+permalink: blog/2017-08-09-two-girls-meet-in-shanghai
+current: blog
+author: BansheeBandits
+twitter: BansheeBandits
+categories:
+- student-posts
+---
+
+## Who are we?! How did we come together?! 
+
+![image](/img/blog/2017/bansheebandits-liz.jpg)
+<div align="center" class="image-credits">Shelby and Lillian with Public Lab's Director of Community Development, Liz Barry (Photo via Shelby Firebaugh)</div>
+
+[Shelby](https://github.com/shelbyfirebaugh) is a rising senior at New York University Tandon School of Engineering working towards a B.S. in Integrated Digital Media. [Lillian](https://github.com/lilliankorinek)--originally from Surprise, Arizona--is pursuing a B.S. in Integrated Media Arts with minors in Public Policy and Nonprofit Management at New York University Shanghai. Shelby and Lillian met in the media lab at NYU Shanghai when Shelby was studying away with NYU at the Shanghai campus. They bonded initially when they were partnered together in “Talking Fabrics,” a creative seminar that covered the application of soft circuits through the physical construction of products.  They stitched together a vest that responded to physical touch using sensors in the Arduino, and have been bonded ever since.
+
+![image](/img/blog/2017/bansheebandits-publiclab.png)
+<div align="center" class="image-credits">Photo via <a href="https://twitter.com/publiclab">Public Lab on Twitter</a></div>
+
+## Why did we choose Public Lab?!?
+
+[Public Lab](https://publiclab.org/home) promotes a hands-on, do-it-yourself ethos--one that supports the concept of exploration, in which members of specific communities across the globe can participate in the collection of environmental data, through DIY methods explained via the Public Lab website. The monumental idea promoted by Public Lab is that technical development and real-life applications can be derived through data experiments conducted by everyday people. 
+
+If government chooses to turn away from solving environmental issues, then it becomes even more important to face these issues in a grassroots fashion. There is power in numbers, and we believe that an organization like Public Lab will be an essential tool in educating and empowering the public, as well as supporting those groups who need help.
+
+Both Lillian and Shelby have an interest in open-source projects, and believe that promoting community participation is not only an efficient way to gather data, but also encourages individuals to become involved regardless of formal experience. We have both spent an extended amount of time in Shanghai, and have witnessed first-hand the damage that industrialization can cause to our environment.
+
+## Meet the team
+
+### **Katie Adee** *Coach*
```

Comment:
Also, just to look it little bit perfect,can you please add comma(,) over here? e.g Katie Adee, Coach